From e1c3ecea26f7fb2d856d1592af0dda5773a8fee3 Mon Sep 17 00:00:00 2001
From: Joachim Schleicher <j.schleicher@pilz.de>
Date: Fri, 14 Dec 2018 09:31:39 +0000
Subject: [PATCH] disable deprecation warnings for the current build

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 87808f2..759b798 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,6 +33,7 @@ find_package(LAPACK REQUIRED)
 
 add_library(${IKFAST_LIBRARY_NAME} src/prbt_manipulator_ikfast_moveit_plugin.cpp)
 target_link_libraries(${IKFAST_LIBRARY_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES} ${LAPACK_LIBRARIES})
+target_compile_options(${IKFAST_LIBRARY_NAME} PRIVATE -Wno-deprecated-declarations)
 
 install(TARGETS ${IKFAST_LIBRARY_NAME} LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
 
-- 
2.7.4

